### ISSUE TITLE
encrypted ci_session

### ISSUE THREAD

**Issue by huxuan** [CAPU-ENG/CAPUrace#219]
We need it encrypted.

**Comment by JeldorPKU** [comment]
We already have `mcrypt` extension on the server side, so the `ci_seesion` is encrypted. Shall we close this one? @huxuan @QwertyJack

**Comment by huxuan** [comment]
Yep, I think mcrypt extension should meet our needs. @QwertyJack , can you help to confirm whether the functionality actually works?

**Comment by JeldorPKU** [comment]
I have just confirmed that it is working. The proof is the following screenshot. 
<img width="896" alt="screenshot 2017-03-08 00 12 19" src="https://cloud.githubusercontent.com/assets/8514953/23665651/0c4ba3de-0394-11e7-9ddf-b34fd144cb96.png">

**Comment by QwertyJack** [comment]
这。。才看到
上次重启就是为了装mcrypto
